apps/bandwagon/forms.py

```diff
@@ -143,6 +145,12 @@ def __init__(self, *args, **kw):
                 self.instance.type in amo.COLLECTION_SPECIAL_SLUGS):
             del self.fields['slug']
 
+    def clean(self):
+        if self.cleaned_data['your_name']:
+            raise forms.ValidationError(
```

Comment:
I'd actually even raise a BadRequest or something in case it's filled out - a validation error was just the simplest way to go and avoid special-case handling in the view.